googletest/test/production.cc

```diff
@@ -30,6 +30,6 @@
 //
 // This is part of the unit test for gtest_prod.h.
 
-#include "production.h"
+#include "test/production.h"
```

Comment:
How is this related to the bazel change? Please revert and just leave the bazel related changes.